File: Java_Design-Patterns/src/adapter/Adapter.java
package adapter;

public class Adapter {

    public static void main(String[] args) {
        Motor motor = new MotorGastalon();
        motor.encender();
        motor.acelerar();
        motor.apagar();

        motor = new MotorEconomico();
        motor.encender();
        motor.acelerar();
        motor.apagar();

        motor = new MotorElectricoAdapter();
        motor.encender();
        motor.acelerar();
        motor.apagar();
    }
}

File: Python_Design-Patterns/adapter/Adapter.py
from MotorEconomico import *
from MotorElectricoAdapter import *
from MotorGastalon import *

class Adapter():

    def main(): 

        motor = MotorGastalon()
        motor.encender()
        motor.acelerar()
        motor.apagar()

        motor = MotorEconomico()
        motor.encender()
        motor.acelerar()
        motor.apagar()

        motor = MotorElectricoAdapter()
        motor.encender()
        motor.acelerar()
        motor.apagar()

    if __name__== "__main__":
        main()